portfolio/src/main/webapp/script.js

```diff
@@ -79,7 +79,10 @@ function fetchBlobstoreUrlAndShowForm() {
         const msgListElement = document.getElementById('messages-container');
         msgListElement.innerHTML = '';
         for(i = 0; i < messages.length ; i++){
-            msgListElement.appendChild(createListElement(messages[i]));
+            // Split the comment and image url into two Strings 
+            var comm = messages[i].split("img:");
```

Comment:
While splitting the long string seems to work at first glance, there might be cases where this will fail. For example, what if the user includes the string "img:" in their comment. Like a comment like "The cat looks good in that img:) Thanks for sharing." will break the parsing. 

We should instead return an array of JSON objects from the server so that we can use message[i].comment and message[i].img to refer to the comment and the image. Please refer to my last comment on the previous PR.

Happy to chat over Google Chat if that helpful.

Reply:
I created a Messages java class with parameters( name comment and image) and added that to the array of messages meant for the json. :)